cpp/include/cudf/io/parquet.hpp

```diff
@@ -552,24 +568,22 @@ class parquet_writer_options {
   /**
    * @brief Sets the maximum row group size, in bytes.
    */
-  void set_row_group_size_bytes(size_t size_bytes)
-  {
-    CUDF_EXPECTS(
-      size_bytes >= 512 * 1024,
-      "The maximum row group size cannot be smaller than the page size, which is 512KB.");
-    _row_group_size_bytes = size_bytes;
-  }
+  void set_row_group_size_bytes(size_t size_bytes) { _row_group_size_bytes = size_bytes; }
 
   /**
    * @brief Sets the maximum row group size, in rows.
    */
-  void set_row_group_size_rows(size_type size_rows)
-  {
-    CUDF_EXPECTS(
-      size_rows >= 5000,
-      "The maximum row group size cannot be smaller than the page size, which is 5000 rows.");
-    _row_group_size_rows = size_rows;
-  }
+  void set_row_group_size_rows(size_type size_rows) { _row_group_size_rows = size_rows; }
+
+  /**
+   * @brief Sets the maximum page size, in bytes.
+   */
+  void set_target_page_size_bytes(size_t pgsz_bytes) { _target_page_size_bytes = pgsz_bytes; }
```

Comment:
These arguments can be named the same as `set_row_group_size_bytes` above. The different function name makes it clear that this is a page size.
```suggestion
  void set_target_page_size_bytes(size_t size_bytes) { _target_page_size_bytes = size_bytes; }
```